wrappers/python/c3py/c3py.py

```diff
@@ -327,24 +336,33 @@ def build_approximation(self, f, fargs, init_rank, verbose, adapt,
 
         self._free_approx_params(c3a, onedopts, low_opts, optnodes)
         c3.fwrap_destroy(fw)
-
-    def build_data_model(self, ndata, xdata, ydata, alg="AIO", obj="LS", verbose=0,
+        
+    def build_model(self, xdata, ydata, LB, UB, nparam=2, init_rank=5, basis="kernel",
```

Comment:
Do not change any functionality of this function, that would lead to breaking of backwards compatability. I am not sure what the purpose of changing the name is. Instead, take out the parts that you want to generalize and create a new function